docker/Dockerfile

```diff
@@ -27,14 +27,20 @@ COPY LICENSE.md .
 COPY CHANGELOG.md .
 COPY COPYING .
 COPY README.md .
+COPY test/ test/
```

Comment:
Hm Id like to avoid this if possible. Not sure how big the test dir is, but it's the vast majority of repo size. Not sure if there's a good way other than another dockerfile..